segment_distance_threshold: 0.012 #0.017
change_segment_distance_threshold: 0.03
cluster_tolerance: 0.05 #0.05
min_cluster_size: 5
max_detection_threshold: 1.5
min_detection_threshold: 0.48
tracking_threshold: 0.4
query_threshold: 0.5
publish_obstacles: true
only_blind_area: false
show_inlier_pc: true
blind_area:
- [1.64,0.594,1.68,-0.082,2.14,-0.109,2.15,0.539]
- [2.66,-0.595,1.22,-0.593,1.22,-1.07,2.7,-1.1]
- [2.69,-2.1,2.69,-2.6,3.2,-2.61,3.16,-2.13]
- [2.21,-3.13,2.25,-3.64,1.65,-3.66,1.7,-3.16]
- [1.17,-2.63,0.185,-2.65,0.197,-3.12,1.21,-3.11]
- [1.17,-2.6,0.179,-2.66,0.184,-2.18,0.606,-2.09,0.662,-1.6,1.16,-1.57]

dead_area:
- [2.61,-2.11,2.63,-2.11,2.69,-2.61,2.75,-2.11]
- [2.66,-0.813,2.67,-0.692,2.53,-0.811,2.53,-0.47]
- [1.17,-0.575,-0.43,-0.644,-0.384,-0.989,1.17,-0.979]
- [1.17,-0.693,1.17,-2.6,1.11,-2.6,1.11,-0.693]
- [-0.32,-1.0,-0.281,-2.67,-0.417,-2.66,-0.417,-1.0]
